CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+<a name="0.21.7"></a>
+### 0.21.7 (2022-12-10)
+
+This release addresses bugs related to server response 
+parsing (thank you to the [OSS-Fuzz project](https://github.com/google/oss-fuzz) for identifying this issue) and 
+`MultiplexedConnection` error handling (thank you to @kvcache for identifying the issue
+and providing a reproducible example).
```

Comment:
I think all the issue-specific commentary here (referring to OSS-Fuzz, thanking kvcache) should move into the bullet points below, keeping this sorter and more focused.